style.css

```diff
@@ -264,16 +270,230 @@ body {
+  gap: 10px;
   display: flex;
 }
 
-.m-l {
-  margin-left: 20px;
-}
-
 .language {
   display: flex;
   justify-content: space-between;
   padding-right: 60px;
 }
+
+@media only screen and (min-width: 768px) {
+  .section-a-container {
+    background-image: url("pimages/header-background-mobile.png");
+    background-position: center;
+    background-size: 100% 90%;
+    background-repeat: no-repeat;
+    display: flex;
+    padding: 0 14em;
+    justify-content: center;
+    align-items: start;
+    flex-direction: column;
+    text-align: start;
+  }
+
+  .navbar-content img {
+    display: none;
+  }
+
+  .navbar {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .nav_link {
+    display: block;
+  }
+
+  .btn {
+    margin-right: -9px;
+  }
+
+  .nav_link ul {
+    display: flex;
+    list-style: none;
+    margin-top: 50px;
+    margin-right: 50px;
+  }
+
+  .nav_link ul a {
+    text-decoration: none;
+    color: black;
+    padding: 10px;
+    font-size: 15px;
+  }
+
+  .desktop {
+    display: grid;
+    grid-template-columns: 1fr;
+    align-items: center;
+    justify-content: center;
+    background-color: #f7f7f9;
+    padding-bottom: 5%;
+  }
+
+  .container-b img {
+    width: 100%;
+    height: 100%;
+  }
+
+  .container-b {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+    gap: 2%;
+    padding: 1rem 0;
+    border-radius: 24px;
+  }
+
+  .container-b > * {
+    flex-basis: calc(50% - 20px);
+  }
+
+  .ruby {
+    padding: 4px 20px;
+    margin-right: -1px;
+  }
+
+  .content {
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+    width: max-content;
+  }
+
+  .content ul {
+    width: max-content;
+    font-size: 18px;
+    color: #6b778c;
+  }
+
+  .content p {
+    /* margin-top: -20px; */
+    font-size: 16px;
+  }
+
+  .container-c {
+    display: grid;
+    grid-template-columns: 1.4fr 2fr;
+  }
+
+  .about-div {
+    padding-left: 50px;
+  }
+
+  .languages {
+    padding-top: 20px;
+    padding-left: 80px;
+    padding-right: 100px;
+  }
+
+  .wrapper {
+    display: flex;
+  }
+
+  .skills {
+    display: flex;
+    flex-direction: column;
+    width: 122px;
+    align-items: flex-start;
+    text-align: start;
+  }
+
+  .last-btn {
+    width: max-content;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .container-b {
+    max-width: 960px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  .container-b > * {
+    flex-basis: calc(33.333% - 20px);
+  }
+
+  .footer-btn {
+    display: flex;
+    margin-left: auto;
+    margin-right: auto;
+  }
+}
+
+footer {
+  background-color: #6070ff;
+  background-image: url(pimages/footer.png);
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: right;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 100px 0 0 0;
+  padding: 4rem 1rem;
+  gap: 3rem;
+  margin-top: 1rem;
+}
```

Comment:
Kindly update the styling of your contact section to match the figma design better🙏

Your design:
<img width="509" alt="image" src="https://github.com/Ejikeokegbe/Mobile-First-portifolio/assets/37116322/6de8df39-5dec-407f-a896-94e7770e485d">

Figma design:
<img width="492" alt="image" src="https://github.com/Ejikeokegbe/Mobile-First-portifolio/assets/37116322/87281772-e7d2-4088-8ab9-3c8beb4f7ba2">

A few things to note:
- Please resize the background image so it matches the figma design
- Please remove the border radius on the input fields on the desktop version